Formula/suil.rb

```diff
@@ -19,10 +21,16 @@ class Suil < Formula
 
   depends_on "pkg-config" => :build
   depends_on "gtk+"
+  depends_on "gtk+3"
   depends_on "lv2"
+  depends_on "qt@5"
+
+  patch :p1, :DATA
```

Comment:
```suggestion
  patch do
    url "https://gitlab.com/lv2/suil/-/commit/33ea47e18ddc1eb384e75622c0e75164d351f2c0.patch"
    sha256 "10dddc02f0a61f03babb4d9692a63aaa2dc9e66e364a2c3098ec5710822002fd"
  end
```
Then you can delete the embedded patch below. Please include a brief comment on what this patch is for and when it can be removed.